products/compute/api.yaml

```diff
@@ -904,14 +883,6 @@ objects:
           'projects/{{project}}/global/backendServices/{{name}}/setSecurityPolicy'
         description: |
           The security policy associated with this backend service.
-      - !ruby/object:Api::Type::ResourceRef
-        name: 'region'
```

Comment:
This won't be a problem for this resource, but something we should think about:

On some providers, removing a field could actually trigger user-facing issues. I'm not worried about this for Ansible at the moment. But, would we need to formally "deprecate" this option on the TF BackendService resource?

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/1689/files#diff-fb4f76e7d870258668a3beac48bf164cL911 makes this a non-issue, and changes like this requiring deprecations is something we're aware of + handle in Terraform. For example: https://github.com/GoogleCloudPlatform/magic-modules/pull/1685#discussion_r278716572